_appmap/env.py

```diff
@@ -46,6 +46,7 @@ def __init__(self, env=None, cwd=None):
 
         self._configure_logging()
         self._enabled = self._env.get("APPMAP", "").lower() != "false"
+        self._enabled_static_request_recordings = self._env.get("APPMAP_ENABLED_STATIC_REQUEST_RECORDINGS", "").lower() != "false"
```

Comment:
Maybe `APPMAP_RECORD_STATIC_REQUESTS` instead? That would be somewhat more consistent with other env vars.

Reply:
@apotterri I had thought of it at first, but then thought it might interfere with https://github.com/getappmap/appmap-python/blob/master/_appmap/env.py#L18